TaskModules/powershell/TaskModuleSqlUtility/SqlPackageOnTargetMachines.ps1

```diff
@@ -448,16 +448,6 @@ function Get-SqlPackageCmdArgs
     return $sqlPkgCmdArgs
 }
 
-function Escape-SpecialChars
-{
-    param(
-        [string]$str
```

Comment:
It will continue to consume module being mentioned there which was being uploaded there. When we change version there at that time we need to remove this method there as well. Should I create PR for that repo also?